client/src/Movies/MovieList.js

```diff
@@ -34,21 +36,8 @@ export default class MovieList extends Component {
 function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
-    <div className="movie-card">
-      <h2>{title}</h2>
-      <div className="movie-director">
-        Director: <em>{director}</em>
-      </div>
-      <div className="movie-metascore">
-        Metascore: <strong>{metascore}</strong>
-      </div>
-      <h3>Actors</h3>
-
-      {stars.map(star => (
-        <div key={star} className="movie-star">
-          {star}
-        </div>
-      ))}
-    </div>
+    <Link to={`/moveis/${movie.id}`}>
```

Comment:
Also, the MovieDetails function does not know anything about what `movie.id` is because of the scope of the function. I think I know why you did this. Although this is what you need, I would move this someplace else. I'll show you where.